graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/SearchDTO.java

```diff
@@ -69,7 +71,11 @@ public abstract static class Builder {
         public abstract String id();
 
         @JsonProperty
-        public abstract Builder queries(Set<QueryDTO> queries);
+        public abstract Builder queries(LinkedHashSet<QueryDTO> queries);
```

Comment:
I understand the usage/creation of the Sets as LinkedHashSets but is it necessary to change the type/interface here?